docs/source/orb/cli.md

```diff
@@ -90,6 +90,31 @@ did create --domain https://orb1.com --publickey-file ./publickeys.json --servic
 ]
 ```
 
+## Resolve
+This command used for resolving DID.
+
+### Usage
+```
+did resolve [flags]
+```
+
+### Flags
+* `domain` _[string]_ - URL to the Orb domain.
+* `sidetree-url-resolution` _[array|string]_ - Array of one or more Sidetree URLs resolution.
```

Comment:
It would be good to have some additional description (and differences) of domain and sidetree-url-resolution for the purpose of resolution.